### ISSUE TITLE
Implement function for URL validation function

### ISSUE THREAD

**Issue by C4r50nC** [C4r50nC/email-handler#1]
Add function to background.js based on the pattern described in the design doc.
- input - currentURL: *String*
- return - true/false: *Boolean*

**Comment by Neru111** [comment]
```
// Function to check if the current URL matches the Gmail URL
function checkURL(currentURL) {
  // Define the pattern for Gmail URLs
  const gmailURLPattern = /^https:\/\/mail\.google\.com\/mail\/u\/\d+\/.*#inbox\/[A-Za-z0-9]+$/; 
  // Check if the current URL matches the pattern
  return gmailURLPattern.test(currentURL);
}
```

**Comment by C4r50nC** [comment]
resolved in #3
